docs/demo_nojs.js

```diff
@@ -0,0 +1,211 @@
+export const popup_demo = `
+  <style>
+    .hamburger { 
+      font-size: 3em; 
```

Comment:
it's definitely preferable to not have the css all through each of these demos, html samples, etc. it's also much more legible (at a certain point, not universally) to have some of it written in scss and then compiled into one `nojavascript.css` file. [[WEB-156](https://bulibrary.atlassian.net/browse/WEB-162)]